File: SemantikosClientesWS/ClienteJava/src/cl/minsal/semantikos/ws/shared/DescripcionPerfectMatch.java

package cl.minsal.semantikos.ws.shared;

import javax.xml.bind.annotation.XmlAccessType;
import javax.xml.bind.annotation.XmlAccessorType;
import javax.xml.bind.annotation.XmlType;


/**
 * <p>Java class for DescripcionPerfectMatch complex type.
 * 
 * <p>The following schema fragment specifies the expected content contained within this class.
 * 
 * <pre>
 * &lt;complexType name="DescripcionPerfectMatch">
 *   &lt;complexContent>
 *     &lt;restriction base="{http://www.w3.org/2001/XMLSchema}anyType">
 *       &lt;sequence>
 *         &lt;element name="idConcepto" type="{http://www.w3.org/2001/XMLSchema}string" minOccurs="0"/>
 *         &lt;element name="idDescripcion" type="{http://www.w3.org/2001/XMLSchema}string" minOccurs="0"/>
 *         &lt;element name="termino" type="{http://www.w3.org/2001/XMLSchema}string" minOccurs="0"/>
 *         &lt;element name="tipoDescripcion" type="{http://www.w3.org/2001/XMLSchema}string" minOccurs="0"/>
 *         &lt;element name="nombreCategoria" type="{http://www.w3.org/2001/XMLSchema}string" minOccurs="0"/>
 *         &lt;element name="terminoPreferido" type="{http://www.w3.org/2001/XMLSchema}string" minOccurs="0"/>
 *       &lt;/sequence>
 *     &lt;/restriction>
 *   &lt;/complexContent>
 * &lt;/complexType>
 * </pre>
 * 
 * 
 */
@XmlAccessorType(XmlAccessType.FIELD)
@XmlType(name = "DescripcionPerfectMatch", propOrder = {
    "idConcepto",
    "idDescripcion",
    "termino",
    "tipoDescripcion",
    "nombreCategoria",
    "terminoPreferido"
})
public class DescripcionPerfectMatch {

    protected String idConcepto;
    protected String idDescripcion;
    protected String termino;
    protected String tipoDescripcion;
    protected String nombreCategoria;
    protected String terminoPreferido;

    /**
     * Gets the value of the idConcepto property.
     * 
     * @return
     *     possible object is
     *     {@link String }
     *     
     */
    public String getIdConcepto() {
        return idConcepto;
    }

    /**
     * Sets the value of the idConcepto property.
     * 
     * @param value
     *     allowed object is
     *     {@link String }
     *     
     */
    public void setIdConcepto(String value) {
        this.idConcepto = value;
    }

    /**
     * Gets the value of the idDescripcion property.
     * 
     * @return
     *     possible object is
     *     {@link String }
     *     
     */
    public String getIdDescripcion() {
        return idDescripcion;
    }

    /**
     * Sets the value of the idDescripcion property.
     * 
     * @param value
     *     allowed object is
     *     {@link String }
     *     
     */
    public void setIdDescripcion(String value) {
        this.idDescripcion = value;
    }

    /**
     * Gets the value of the termino property.
     * 
     * @return
     *     possible object is
     *     {@link String }
     *     
     */
    public String getTermino() {
        return termino;
    }

    /**
     * Sets the value of the termino property.
     * 
     * @param value
     *     allowed object is
     *     {@link String }
     *     
     */
    public void setTermino(String value) {
        this.termino = value;
    }

    /**
     * Gets the value of the tipoDescripcion property.
     * 
     * @return
     *     possible object is
     *     {@link String }
     *     
     */
    public String getTipoDescripcion() {
        return tipoDescripcion;
    }

    /**
     * Sets the value of the tipoDescripcion property.
     * 
     * @param value
     *     allowed object is
     *     {@link String }
     *     
     */
    public void setTipoDescripcion(String value) {
        this.tipoDescripcion = value;
    }

    /**
     * Gets the value of the nombreCategoria property.
     * 
     * @return
     *     possible object is
     *     {@link String }
     *     
     */
    public String getNombreCategoria() {
        return nombreCategoria;
    }

    /**
     * Sets the value of the nombreCategoria property.
     * 
     * @param value
     *     allowed object is
     *     {@link String }
     *     
     */
    public void setNombreCategoria(String value) {
        this.nombreCategoria = value;
    }

    /**
     * Gets the value of the terminoPreferido property.
     * 
     * @return
     *     possible object is
     *     {@link String }
     *     
     */
    public String getTerminoPreferido() {
        return terminoPreferido;
    }

    /**
     * Sets the value of the terminoPreferido property.
     * 
     * @param value
     *     allowed object is
     *     {@link String }
     *     
     */
    public void setTerminoPreferido(String value) {
        this.terminoPreferido = value;
    }

}

File: SemantikosClientesWS/clientePHP/src/Semantikos/ClientBundle/API/DescripcionPerfectMatch.php
<?php

namespace Semantikos\ClientBundle\API;

class DescripcionPerfectMatch
{

    /**
     * @var string $idConcepto
     */
    protected $idConcepto = null;

    /**
     * @var string $idDescripcion
     */
    protected $idDescripcion = null;

    /**
     * @var string $termino
     */
    protected $termino = null;

    /**
     * @var string $tipoDescripcion
     */
    protected $tipoDescripcion = null;

    /**
     * @var string $nombreCategoria
     */
    protected $nombreCategoria = null;

    /**
     * @var string $terminoPreferido
     */
    protected $terminoPreferido = null;

    
    public function __construct()
    {
    
    }

    /**
     * @return string
     */
    public function getIdConcepto()
    {
      return $this->idConcepto;
    }

    /**
     * @param string $idConcepto
     * @return DescripcionPerfectMatch
     */
    public function setIdConcepto($idConcepto)
    {
      $this->idConcepto = $idConcepto;
      return $this;
    }

    /**
     * @return string
     */
    public function getIdDescripcion()
    {
      return $this->idDescripcion;
    }

    /**
     * @param string $idDescripcion
     * @return DescripcionPerfectMatch
     */
    public function setIdDescripcion($idDescripcion)
    {
      $this->idDescripcion = $idDescripcion;
      return $this;
    }

    /**
     * @return string
     */
    public function getTermino()
    {
      return $this->termino;
    }

    /**
     * @param string $termino
     * @return DescripcionPerfectMatch
     */
    public function setTermino($termino)
    {
      $this->termino = $termino;
      return $this;
    }

    /**
     * @return string
     */
    public function getTipoDescripcion()
    {
      return $this->tipoDescripcion;
    }

    /**
     * @param string $tipoDescripcion
     * @return DescripcionPerfectMatch
     */
    public function setTipoDescripcion($tipoDescripcion)
    {
      $this->tipoDescripcion = $tipoDescripcion;
      return $this;
    }

    /**
     * @return string
     */
    public function getNombreCategoria()
    {
      return $this->nombreCategoria;
    }

    /**
     * @param string $nombreCategoria
     * @return DescripcionPerfectMatch
     */
    public function setNombreCategoria($nombreCategoria)
    {
      $this->nombreCategoria = $nombreCategoria;
      return $this;
    }

    /**
     * @return string
     */
    public function getTerminoPreferido()
    {
      return $this->terminoPreferido;
    }

    /**
     * @param string $terminoPreferido
     * @return DescripcionPerfectMatch
     */
    public function setTerminoPreferido($terminoPreferido)
    {
      $this->terminoPreferido = $terminoPreferido;
      return $this;
    }

}
